Resources/Prototypes/Entities/Mobs/NPCs/bear.yml

```diff
@@ -32,13 +43,12 @@
     allowedStates:
       - Alive
       - Dead
-  - type: MobThresholds
```

Comment:
I don't think the modifier changes justify changing their alive thresholds from 0-150 to 0-50

Reply:
All of the changes were requested by ODJ as part of the overall damage overhaul. I went with the values he asked for.

Reply:
It should be 0-100 though, it will inherit this
>
  - type: MobThresholds
    thresholds:
      0: Alive
      50: Critical
      100: Dead

Reply:
critical may as well be dead for an npc, no one is healing them

Reply:
Fair, but it would have been 100: Crit before, 150: dead. So, it's a nerf of 50 not 100.

Reply:
Maybe Crit should be 75?

Reply:
OK, I talked with ODJ and found where the confusion was. Will fix this soon.